src/common/net.cc

```diff
@@ -337,6 +370,41 @@ namespace Pistache
 
     void Address::init(const std::string& addr)
     {
+        // Handle unix domain addresses separately.
+        if (isUnixDomain(addr))
+        {
+            struct sockaddr_un unAddr;
+            unAddr.sun_family = AF_UNIX;
+
+            // See unix(7) manual page; distinguish among unnamed, abstract,
+            // and pathname socket addresses.
+            auto size = std ::min(addr.size(), sizeof unAddr.sun_path);
```

Comment:
```suggestion
            const auto size = std::min(addr.size(), sizeof(unAddr.sun_path));
```

Reply:
Fixed the space before :: and the missing auto.  But the formatting for sizeof triggers one of my pet peeves.  sizeof is not a function; it's a keyword operator and keywords should be followed by a space.  Omitting parens around its operand make it clear that the operand is an object as opposed to a type.

Having said that, if your style guide says that it should be formatted as you've suggested, I'll do so.